Oppgave 2.ipynb

```diff
@@ -0,0 +1,2147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    X_data = data.drop(columns = ['Gender'])
</pre><p>Det er ikke meningen å fjerne kjønns-kolonnen her. Du skal gjøre analyse basert på kjønn. F.eks sjekke ytelse for menn vs kvinner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bsingstad/DMBA2024/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>